src/fraclmul.adoc

```diff
@@ -0,0 +1,175 @@
+[appendix]
+== Fractional Lmul example
```

Comment:
I would suggest to change this to be a subsection of appendix with the vector assembly code examples, not as a standalone extra appendix. And it should follow the vector assembly code examples appendix, not the vector calling convention appendix.

Reply:
I've gone ahead and moved it into the Vector Assembly Code Examples appendix.